### ISSUE TITLE
Code of Conduct

### ISSUE THREAD

**Issue by brewcore** [TechLancaster/community#2]
Let's start discussion on a community accepted code-of-conduct here.

**Comment by gdtroszak** [comment]
[Code for America's code of conduct](https://github.com/codeforamerica/codeofconduct) may be a good place to start. Plus it can be forked :)

**Comment by brewcore** [comment]
Yeah, that looks promising.  CPOSC.org has a decent one too with a link to some other resources.  http://cposc.org/code-of-conduct/

**Comment by brisayswhaat** [comment]
Here's another, this touches a lot on harassment & unacceptable behaviors in a setting: https://www.girldevelopit.com/code-of-conduct 

This is another good resource: http://www.ashedryden.com/blog/codes-of-conduct-101-faq

**Comment by rjfisher** [comment]
I really like the [GDI CoC](https://www.girldevelopit.com/code-of-conduct) as Bri points out it defines more about bad behavior and consequences.  The only thing I _don't_ like is that we don't really have a governing body.  If there was harassment at a meeting who would one lodge a complaint with?  The leader of that meeting?  Are the separate meetings to have their own processes or interpretations or harassment?

Should these meetups that are sort of loosely affiliated have some sort of loose, rotating membership group that would be there to listen to claims of harassment and respond appropriately?

At this point it might all be hypothetical but I don't want meteor-mondays to be MY meetup.  I want it to be a meetup.  I don't want to be the dictator of my own little fiefdom anymore than I want anyone ELSE to be the dictator of their own little fiefdom.

**Comment by brisayswhaat** [comment]
@rjfisher That's a really good point. I think if we went on a meetup by meetup basis, it would whoever the core organizer(s) are and it would be up to them to enforce it and interpret it. Obviously not everyone has to since we're not an organization of any kind and there's no way to govern it other than karma and having a conscience.

**Comment by rjfisher** [comment]
@brisayswhaat I'm on board with that but as we umbrella these meetups under a sort of single flag. "the Tech Lancaster" flag (when do we all get t-shirts?) I think it would be smart that a single meetup not necessarily harm the tone of openness and acceptance of the meetups in general by being a single bad apple.

If the primary organizers get to be lords of their own fiefdoms it is important that the greater collaboration of groups have some method to not let one group get out of hand.

Hypothetically if MM were to be building a game that allowed you to punch women or children and I was totally on board with that vision I can't imagine the rest of the groups would want to associate with MM anymore.  At some point the organizers should have some understand of how to DROP meetups if it comes to that or otherwise encourage openness.

I guess I'm advocating for SOME sort of general ruleset that'll justify getting dropped from the community if one fiefdom gets out of hand.

**Comment by brewcore** [comment]
I've started laying some groundwork for this in the Readme file in the repo.
